include/alpaka/math/MathUniformCudaHipBuiltIn.hpp

```diff
@@ -11,31 +11,6 @@
 
 #if defined(ALPAKA_ACC_GPU_CUDA_ENABLED) || defined(ALPAKA_ACC_GPU_HIP_ENABLED)
 
-#    include <alpaka/core/BoostPredef.hpp>
-
-#    if defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
-#        include <cuda_runtime.h>
-#        if !BOOST_LANG_CUDA
-#            error If ALPAKA_ACC_GPU_CUDA_ENABLED is set, the compiler has to support CUDA!
-#        endif
-#    endif
-
-#    if defined(ALPAKA_ACC_GPU_HIP_ENABLED)
-
-#        if BOOST_COMP_NVCC >= BOOST_VERSION_NUMBER(9, 0, 0)
-#            include <cuda_runtime_api.h>
-#        else
-#            if BOOST_COMP_HIP
-#                include <hip/math_functions.h>
-#            else
-#                include <math_functions.hpp>
-#            endif
-#        endif
-#        if !BOOST_LANG_HIP
-#            error If ALPAKA_ACC_GPU_HIP_ENABLED is set, the compiler has to support HIP!
-#        endif
-#    endif
-
```

Comment:
I think this whole piece should not have been here at all. All the header files do the check, and this file by itself does not add anything. So here i just removed it

Reply:
Fine for me!